approaches/bulk-data-loading/uni_configuration/schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://vendia.net/schemas/examples/inventory-management-system.json",
+    "title": "Inventory Management System",
+    "description": "All inventory data for a Retailer",
+    "type": "object",
+    "properties": {
+      "Inventory": {
+        "description": "Inventory",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "Item name",
+              "type": "string"
+            },
+            "number": {
+              "description": "Item number",
+              "type": "string"
+            },
+            "quantity": {
```

Comment:
In terms of sample code it may be useful to add another level of nesting in the schema and demonstrate how it can flattened in CSV and mapped to a nested property in the schema.

Reply:
Done.  Added a `supplier` field to the schema, which is an object that contains to fields itself.